llvm/lib/Transforms/Instrumentation/MemProfiler.cpp

```diff
@@ -171,6 +171,10 @@ static cl::opt<std::string>
                                  cl::desc("The default memprof options"),
                                  cl::Hidden, cl::init(""));
 
+static cl::opt<bool> UndriftProfile("memprof-undrift-profile",
```

Comment:
Can you change this flag to include "stale-profile" in the name so that it's easier to find related work in the LLVM code base? For example, we have "salvage-stale-profile" (for Sample PGO) and "infer-stale-profile" for BOLT. This flag could be "memprof-salvage-stale-profile" or something similar. Wdyt?

Also undrift is not a well known term in LLVM (all the references are in MemProf) so we should spell out what we mean in the description.

Reply:
I've renamed the option to `-memprof-salvage-stale-profile`.  Please take a look.  Thanks!